data/json/items/chemicals_and_resources.json

```diff
@@ -787,5 +787,39 @@
     "ammo_type" : "components",
     "container" : "box_small",
     "count" : 20
+   },
+   {
+    "type" : "COMESTIBLE",
```

Comment:
Does it need to be comestible for it to work?
I'm pretty sure that it could be just generic nowadays. No need to supply it with extra data that will only make it pollute the eat menu.

Reply:
Changing it to
```
{
    "type" : "GENERIC",
    "id" : "lye",
    "name" : "lye",
    "name_plural" : "lye",
    "weight" : 125,
    "color" : "white",
    "container" : "bottle_plastic",
    "symbol" : "~",
    "description" : "This is a liquid form of sodium hydroxide.  It is highly corrosive, handle with care.",
    "price" : 100,
    "volume" : 1,
    "phase" : "liquid"
   }
```
makes the game to crash. I'm doing something wrong?

Reply:
I don't see anything wrong with the definition.
Well, I guess maybe the game isn't ready for non-comestible, non-ammo liquids yet.